ct_optcon/examples/nlocCost_ObstacleConstraint.info

```diff
@@ -12,7 +12,12 @@ intermediateCost
         (0,0) 0
         (1,1) 0
         (2,2) 0
-
+      }
```

Comment:
@hmcm please note that your R matrix MUST be in the intermediate cost, rather the terminal cost. This was the root cause of the improper convergence that we saw in your original example.